krabcake/Makefile.am

```diff
@@ -13,18 +33,25 @@ endif
 
 KRABCAKE_SOURCES_COMMON = kc_main.c
 
+# RUST_EXTRA_LD_FLAGS = -ldl -lrt -lpthread -lgcc_s -lc -lm -lrt -lutil
+# RUST_EXTRA_LD_FLAGS = -ldl -lrt -lpthread -lc -lm -lrt -lutil
+RUST_EXTRA_LD_FLAGS =
```

Comment:
this + its usage below is currently a no-op right?

Reply:
that's right; this is a relic from when I was trying to figure out some linker stuff. (I never really resolved it to my satisfaction, though at least now I have some theories as to what's going on...)